configs/projects/_shared-agent-components.rb

```diff
@@ -62,6 +62,10 @@
 proj.component 'rubygem-fast_gettext'
 proj.component 'rubygem-ffi'
 
+# Note: We won't need to explicitly add 'rubygem-rexml' to agent-runtimes once ruby is upgraded to 3.3.3 or higher (CVE-2024-35176 and CVE-2024-39908).
+# This is because the bundled rexml gem version shipped with ruby itself will be free from these CVEs.
+proj.component 'rubygem-rexml'
```

Comment:
do we want this in both streams?

Reply:
oh, sorry i see that in the commit message. So for ruby 2.7 we want to override the default rexml version.